### DISCUSSION TITLE
Facebook

### DISCUSSION THREAD

**Discussion by leonrvs** [Authenticator-Extension/Authenticator#823] [Q&A]
### Describe the issue

Hi, Facebook is not accepting any codes ; (
Although it was authenticated and worked great for great a while.
I use your extention for facebook business, and with RPA I can do a lot for reporting.

Any idea Why not accepted anymore?

I reauthenticate the extention but when I scaned the QRcode, it will not accept the code of the authenticator....
I used another browser extention GAuth authenticator (google), but that is not accepted either..

### Browser

Chrome

### Browser Version

Version 96.0.4664.110 (Official Build) (64-bit)

### Extension Version

6.3.3

**Comment by mymindstorm** [comment]
Have you tried Menu -> sync clock?

**Comment by leonrvs** [comment]
Adding a new app in Facebook using your authenticator doesn't work at the moment with Facebook. It did work..

Kind regards,
Outlook voor iOS<https://aka.ms/o0ukef> downloaden
________________________________
Van: sandarjue00 ***@***.***>
Verzonden: Monday, December 27, 2021 3:03:51 AM
Aan: Authenticator-Extension/Authenticator ***@***.***>
CC: leonrvs ***@***.***>; Author ***@***.***>
Onderwerp: Re: [Authenticator-Extension/Authenticator] Facebook (Issue #807)


Ok

—
Reply to this email directly, view it on GitHub<https://github.com/Authenticator-Extension/Authenticator/issues/807#issuecomment-1001289270>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADRZAUVADTRLVWPT4SU6XCDUS7CQPANCNFSM5KQFPRIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
